x-pack/solutions/security/plugins/security_solution/common/api/detection_engine/model/rule_schema/common_attributes.schema.yaml

```
@@ -70,6 +70,23 @@ components:
       type: boolean
       description: Determines whether an external/prebuilt rule has been customized by the user (i.e. any of its fields have been modified and diverged from the base value).
 
+    ExternalRuleHasBaseVersion:
+      type: boolean
+      description: Determines whether or not an external/prebuilt rule has its original, unmodified version present when the `is_customized` calculation is performed.
```

```suggestion
      description: Determines whether an external/prebuilt rule has its original, unmodified version present when the `is_customized` calculation is performed.
```
Not sure what 'external' rule means – AFAIK, that's not a term we use in our user docs.

---

It is another name for prebuilt rules, we had already used the naming in the [current docs](https://www.elastic.co/docs/api/doc/kibana/operation/operation-findrules#operation-findrules-200-body-application-json-data-rule_source) for the rule source object so I just kept it the same.